src/node_url.cc

```diff
@@ -131,11 +131,13 @@ namespace url {
   static int ToUnicode(std::string* input, std::string* output) {
     output->reserve(input->length());
     *output = input->c_str();
```

Comment:
> Also: it looks like both the ICU and non-ICU implementations should be able to use `input->data()` instead of `input->c_str()`.  Avoids an unnecessary reallocation.

`data()` is an alias for `c_str()` since C++11. ;)